Sources/BraveWallet/Crypto/Stores/BuyTokenStore.swift

```diff
@@ -57,14 +58,37 @@ public class BuyTokenStore: ObservableObject {
     self.rpcService = rpcService
     self.walletService = walletService
     self.assetRatioService = assetRatioService
-    self.selectedBuyToken = prefilledToken
+    self.prefilledToken = prefilledToken
```

Comment:
probably dont need this property at all as you just need to pass this value to `validatePrefilledToken` which will be called in updateInfo()
Same in send and swap

Reply:
I think it's better if we don't handle this at initialization of the storekeep the initialization clean.